espnet/nets/pytorch_backend/e2e_asr_transducer.py

```diff
@@ -134,6 +134,7 @@ def __init__(self, idim, odim, args):
         self.space = args.sym_space
         self.blank = args.sym_blank
         self.reporter = Reporter()
+        self.beam_size = args.beam_size
```

Comment:
You don't need to handle greedy/beam search selection here. It is already handled in asr and asrinterface modules : 

https://github.com/espnet/espnet/blob/ee0c82d6e74659bd174c90a261135e0901f8b39b/espnet/asr/pytorch_backend/asr.py#L688-L749

https://github.com/espnet/espnet/blob/ee0c82d6e74659bd174c90a261135e0901f8b39b/espnet/nets/asr_interface.py#L66-76

Reply:
Sorry for not explaining it in detail. This code is used to calculate cer and wer for each epoch when training. In other words, the train was slow because only beam search was supported. 
I want to use greedy search when calculating cer / wer on training.(not inference: asr_recog.py) 
So i think we need this. This error also happens in models(e2e_asr_transducer.py) that don't have the recognize_batch function when decoding (inference) after the train ends, setting the decode batch size to 1 or more.(ex. voxforge corpus, in decode_transducer.yaml, batch: 1)

Reply:
Yes sorry, I mistook beamsize for batchsize! I made the review in a rush and I realized it after. Discard my review :)

For adding greedy decode option for cer/wer computation during training I don't see any problem. However you talk about an "error", it should not raise any error, it only not supported yet, right?